ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/SqlLedgerSpec.scala

```diff
@@ -31,6 +32,8 @@ class SqlLedgerSpec
 
   private val ledgerId: LedgerId = LedgerId(Ref.LedgerString.assertFromString("TheLedger"))
 
+  private val namedLogger = NamedLogger(SqlLedgerSpec.super.getClass)
```

Comment:
I'm curious, what's the reason for using `super` here?

Reply:
A mistake - I will change this to this.getClass